zone/exp.cpp

```diff
@@ -1182,8 +1182,8 @@ void Raid::SplitExp(const uint64 exp, Mob* other) {
 
 	if (RuleB(Character, EnableRaidEXPModifier)) {
 		raid_experience = static_cast<uint64>(
-			static_cast<float>(raid_experience) *
-			(1.0f - RuleR(Character, RaidExpMultiplier))
+				static_cast<float>(raid_experience) *
```

Comment:
This seems to be indented in one more level than needbe

Reply:
Converted to single line.